2-copy-of-code/lesson-16/tests/data/cartTest.js

```diff
@@ -136,4 +136,22 @@ describe('test suite: updateDeliveryOption', () => {
     expect(cart[0].deliveryOptionId).toEqual('1');
     expect(localStorage.setItem).toHaveBeenCalledTimes(0);
   });
+
+  it('does nothing if the delivery option does not exist', () => {
+    spyOn(localStorage, 'getItem').and.callFake(() => {

```

Comment:
If you use the `spyOn(localStorage, 'getItem')` inside `beforeEach( )` and also use `spyOn(localStorage, 'getItem')` inside the `it( )` function then you will get this type of error. Because `spyOn( )` was used twice.

Reply:
it means to have spied on (localStorage, 'getItem') twice in your test